en_US/deploy/cluster/create-cluster.md

```diff
@@ -1,140 +1,142 @@
+```bash
+node.name = emqx@s1.emqx.io
+# or
+node.name = emqx@192.168.0.10
+```
 
-   1. leave: let the node leave the cluster
-   2. force-leave: remove the node from the cluster
+You can also override the node name with an environment variable:
 
-   Run the command below on `emqx@s2.emqx.io` to let it leave the cluster:
+```bash
+env EMQX_NODE__NAME='emqx@s1.emqx.io' ./bin/emqx start
+```
 
-   ```bash
-   ./bin/emqx_ctl cluster leave
-   ```
+Repeat the above step for the other node to join the cluster. 
 
-   Or run the command below on `emqx@s1.emqx.io` to remove `emqx@s2.emqx.io` from the cluster:
+Now you have named 2 nodes to join the cluster, `emqx@s1.emqx.io` and `emqx@s2.emqx.io`. You can create a cluster either manually or automatically. 
 
-   ```bash
-   ./bin/emqx_ctl cluster force-leave emqx@s2.emqx.io
-   ```
+## Manual Clustering
+
+### Set Node Discovery Strategy
 
-### Pseudo-distributed cluster
+EMQX supports creating clusters manually and automatically. This section will introduce the manual clustering feature in EMQX. 
 
-For users with only one server, you can use the pseudo-distributed cluster to test the EMQX cluster.
+Set node discovery strategy
 
-After starting the first node, use the following command to start the second node and join the cluster manually. To avoid port conflicts, we need to adjust some listening ports:
+Manual clustering is the method to configure an EMQX cluster by manually specifying which nodes should be part of the cluster. By default, EMQX adopts a manual clustering strategy, which can also be set in `emqx.conf`. Besides, you also need to configure the default 
 
 ```bash
-EMQX_NODE__NAME='emqx2@127.0.0.1' \
-    EMQX_STATSD__SERVER='127.0.0.1:8124' \
-    EMQX_LISTENERS__TCP__DEFAULT__BIND='0.0.0.0:1882' \
-    EMQX_LISTENERS__SSL__DEFAULT__BIND='0.0.0.0:8882' \
-    EMQX_LISTENERS__WS__DEFAULT__BIND='0.0.0.0:8082' \
-    EMQX_LISTENERS__WSS__DEFAULT__BIND='0.0.0.0:8085' \
-    EMQX_DASHBOARD__LISTENERS__HTTP__BIND='0.0.0.0:18082' \
-    EMQX_NODE__DATA_DIR="./data2" \
-./bin/emqx start
-
-./bin/emqx_ctl cluster join emqx1@127.0.0.1
+cluster {
+    ## options: manual | static | dns | etcd | K8s
+    discovery_strategy  =  manual
+}
 ```
 
-## Node discovery and auto clustering
-
-Node discovery allows individual EMQX node with different IPs or locations to be discovered and communicated with each other, and it is a crucial step when creating EMQX clusters.
+This approach provides users with greater control and flexibility over the cluster configuration and more security as you can restrict which nodes can join the cluster. 
 
-EMQX's autocluster feature is based on the [Ekka](https://github.com/emqx/ekka) library, the cluster management library developed for Erlang/OTP applications, supporting features like Erlang node service discovery, autocluster, network partition autoheal, and autoclean.
+:::tip
+Manual clustering is not supported in the Core-Replica architecture.
+:::
```

Comment:
This issue has since been fixed, so this tip can be removed.

Reply:
removed, thanks